src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
-body {
+$font-family: 'Roboto', sans-serif;
+$main-color: #060b35;
+$second-color: #616070;
+
+body,
+p {
   margin: 0;
+  font-family: $font-family;
+  color: $main-color;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use `0` instead of `0px`. So instead of `margin: 0;` you can simply use `margin: 0;`.